pd2jira_python/Ticket.py

```diff
@@ -91,6 +93,9 @@ def get_labels_field(self):
     def get_priority_field(self):
         return {self.PRIORITY_KEY: self.CONFIGS['priority']}
 
+    def get_components_field(self):
+        return {self.COMPONENTS_KEY: self.CONFIGS['components']}
```

Comment:
can we default to a blank value, so that code does not break, if this config is not provided ?